firestore/firestore_snippets/listen_test.go

```diff
@@ -114,11 +113,14 @@ func TestListenChanges(t *testing.T) {
 	// Add some changes to data in parallel.
 	time.Sleep(time.Second)
 	var pop int64 = 3900000
-	if _, err := client.Collection(collection).Doc("LA").Update(ctx, []firestore.Update{
-		{Path: "population", Value: pop},
-	}); err != nil {
-		log.Fatalf("Doc.Update: %v", err)
-	}
+
+	testutil.Retry(t, 10, 10*time.Second, func(r *testutil.R) {
```

Comment:
Looking at this more closely, I don't think this retry should be necessary since we were not seeing any failures in L120 previously, only at L125 (original line numbers). Increasing the timeout alone should probably be sufficient. Any thoughts @BenWhitehead @tbpg ?

Reply:
+1 -- no need to add this retry if this isn't where the failure was.

Reply:
Ok, got it